cmd/sidecars/kafka.go

```diff
@@ -28,6 +28,11 @@ func (kc KafkaConfig) Broker() string {
 // KafkaToDeployment receives a the KafkaConfig variable as a parameter and returns a
 // SidecarFactory function that is used to subscribe to the sidecarFactory
 func KafkaToDeployment(config KafkaConfig) models.SidecarFactory {
+	// Handles defaults values in case any of the kafkaConfig variables are empty
+	if config.KafkaInsprAddr == "" {
+		config.KafkaInsprAddr = "http://localhost"
```

Comment:
I dont understand the use of "localhost" here, it's just a default in case KafkaInsprAddr is not set? It should not return an error?

Reply:
Yeah, the default should be localhost and should not return an error